### ISSUE TITLE
Cannot read property 'type' of undefined

### ISSUE THREAD

**Issue by hassaans** [braintree/braintree-web-drop-in#471]
### General information

* SDK/Library version: 1.12.0
* Environment: Production
* Browser and OS: 8 events on Chrome 71.0.3578, 1 event on Chrome 70.0.3538. Multiple OS

### Issue description

I have pinpointed the error to this [line](https://github.com/braintree/braintree-web-drop-in/blob/05992be8711535227b739929aa85ba52fbb5587a/src/views/payment-sheet-views/card-view.js#L281)

### Exception (sentry):

```
TypeError: Cannot read property 'type' of undefined
  at h._validateForm(/dist/gen/vendors~xxx.5b77e104a96fa1555efd.prod.js:79:269574)
  at h._sendRequestableEvent(/dist/gen/vendors~xxx.5b77e104a96fa1555efd.prod.js:79:267347)
  at h._onValidityChangeEvent(/dist/gen/vendors~xxx.5b77e104a96fa1555efd.prod.js:79:274476)
  at M.n._emit(/dist/gen/vendors~xxx.5b77e104a96fa1555efd.prod.js:79:291252)
  at M.<anonymous>(/dist/gen/vendors~xxx.5b77e104a96fa1555efd.prod.js:79:276954)
  at MessageEvent.a(/dist/gen/vendors~xxx.5b77e104a96fa1555efd.prod.js:43:183842)
```

Breadcrumbs (sentry):
```
17:47:26
ui.click
div.sc-kcbnda.jOriPb > div.sc-iIHSe.cANhXL > button.sc-htpNat.sc-hcmgZB.iSrXqG
17:47:40
ui.click
div.braintree-option.braintree-option__card > div.braintree-option__label
17:48:27
exception
TypeError: Cannot read property 'type' of undefined
```

**Comment by crookedneighbor** [comment]
Does Does the same thing occur in v1.14.1? What are the steps to reproduce the error?

**Comment by crookedneighbor** [comment]
Closing this due to inactivity.

**Comment by rebendajirijr** [comment]
The issue still persists in `1.18.0`.

![image](https://user-images.githubusercontent.com/3431185/62198562-276bb000-b382-11e9-8ce6-d595811b6bde.png)

**Comment by crookedneighbor** [comment]
@rebendajirijr I assume it's happening at this line: https://github.com/braintree/braintree-web-drop-in/blob/9942c735e21d4adf14ddde617fd82764cd1dfb82/src/views/payment-sheet-views/card-view.js#L293

I still need a reproducible case though so I can be sure. Can you give me steps to cause that error to happen? Do you have a demo site where it happens?

**Comment by crookedneighbor** [comment]
This is fixed on master and will go out in the next release. We'll close this when the next release is out.

**Comment by crookedneighbor** [comment]
Fix is out in This is out in v1.20.2
